i18n/en.json

```diff
@@ -15,5 +15,6 @@
 	"wikibasereconcileedit-editendpoint-qualifiers-references-not-supported": "Qualifiers and References are not currently supported",
 	"wikibasereconcileedit-editendpoint-reconciliation-property-missing-in-statements": "Entity must have at least one statement for the reconciliation Property",
 	"wikibasereconcileedit-editendpoint-invalid-reconciliation-statement-type": "Reconciliation statement must contain a value",
-	"wikibasereconcileedit-reconciliationservice-matched-multiple-items": "Matched $2 Items during reconciliation :(\n\n{{PLURAL:$2|item is|items are}} $1\n"
+	"wikibasereconcileedit-reconciliationservice-matched-multiple-items": "Matched $2 Items during reconciliation :(\n\n{{PLURAL:$2|item is|items are}} $1\n",
+	"wikibasereconcileedit-minimaliteminput-required-property-value-keys": "Statements must have property and value keys"
```

Comment:
Maybe the words “property” and “value” should be message parameters so that we can change them without changing the message… what do you think?

Or maybe this should just be one message about a missing key, and then we check the keys separately in the service, and throw the message with either a “property” or “value” parameter?

(If we don’t go with that, then the `qqq.json` should at least mention that the words “property” and “value” should not be translated, in my opinion.)

Reply:
I would go with the third option. It's always going to be either the property or the value, right? The user can probably figure out which one is wrong.

Reply:
Hmm, `wikibasereconcileedit-minimaliteminput-required-property-value-keys`seems to have disappeared in the branch is this still an issue?

Reply:
Should be back there now.

Reply:
I think this part got accidentally dropped? As far as I can tell, `die-error-handling-2` currently still doesn’t have this message, and also still has `die( 'statements must have property and value keys' )`.

Reply:
Okay, but the branch is still missing any of the “three options” as far as I can tell.

I also feel like the third option is needlessly user-hostile. We already know which of the two keys is missing – we even have separate tests for the two cases – so why should we combine them into one, less specific message?

I think the “second option” (one message for a missing key, with “property” or “value” in an argument) is the best one – we can then also use this message later when the keys change (e.g. we add other ones).

Reply:
The comment was there before in `qqq.json. Must have been lost underway.

Reply:
Yes, but that doesn’t mean I have to like the third option :P